Telemetry/CorePing.swift

```diff
@@ -78,6 +78,16 @@ class CorePing: TelemetryPing {
             "tz": timezoneOffset
         ]
 
+        if let searches = SearchTelemetryFactory.getData(profile.prefs) {
+            out["searches"] = searches
+
+            // Clearing the count here means we'll lose the data if the ping doesn't go through.
```

Comment:
This is the comment I'd move to the spot in the first commit – I feel it's better there because that's the code that, if changed, would change this behavior.